docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,744 @@
+.. _integrations-pytorch-lightning:
+
 Neptune-PyTorch Lightning Integration
 =====================================
 
-The integration enables you to log |PyTorch Lightning| metrics to Neptune.
-
-.. image:: ../_static/images/integrations/pytorchlightning_neptuneml.png
-   :target: ../_static/images/integrations/pytorchlightning_neptuneml.png
+What will you get?
+------------------
+.. image:: ../_static/images/integrations/lightning_basic.gif
```

Comment:
Could you change this to a video -> I think with video you could show/talk more about what you get and what you need to add to do it.  Also, it would be more reusable. 

I suggest:
- 30-60s
- it could start in the code -> then run from terminal -> go to the app (or skip the terminal)
- add an inviting thumbnail -> if you use this loom thing I think we can get a really cool one with your face even!!

Reply:
Agree 101% :) I'll do it, and let you know.

Reply:
Added video